app/server/routes/api.js

```diff
@@ -373,4 +392,46 @@ module.exports = function(router) {
     SettingsController.updateWhitelistedEmails(emails, defaultResponse(req, res));
   });
 
+  /**
+   * [ADMIN/OWNER]
+   * Edits the hack's start time
+   */
+  router.put('/settings/checkInStart', isAdmin, function(req, res) {
+    var checkInOpen = req.body.checkInStart;
+    SettingsController.updateField('checkInOpen',checkInOpen, defaultResponse(req, res));
+  });
+
+  /**
+   * [ADMIN/OWNER]
+   * Edits the hack's allowed team size
+   */
+  router.put('/settings/teamSizeAccepted', isAdmin, function(req, res) {
+    var teamSize = req.body.teamSizeAccepted;
+    SettingsController.updateField('teamSizeAccepted', teamSize, defaultResponse(req, res));
+  });
+
+  /**
+   * [ADMIN/OWNER]
+   * Edits the hack's location
+   */
+  router.put('/settings/hackLocation', isAdmin, function(req, res) {
+    var locationLatitude = req.body.latitude;
+    var locationLongitude = req.body.longitude;
+
+    var hackLocation = {
```

Comment:
This is great, is there any reason why you didn't name the variables `latitude` and `longitude`?

That way you could also do the following and obtain the same result:

`var hackLocation = { latitude, longitude };`